static/app/views/performance/traceDetails/transactionBar.tsx

```diff
@@ -470,6 +474,35 @@ class TransactionBar extends Component<Props, State> {
     );
   }
 
+  renderPerformanceIssues() {
+    const {transaction, barColor} = this.props;
+    if (!isTraceFullDetailed(transaction)) {
+      return null;
+    }
+
+    const rows: React.ReactElement[] = [];
+    const delta = Math.abs(transaction.timestamp - transaction.start_timestamp) || 1;
+    for (let i = 0; i < transaction.performance_issues.length; i++) {
+      const issue = transaction.performance_issues[i];
+      // Use 1 as the difference in the event that startTimestamp === endTimestamp
```

Comment:
I know this is copypasta from above, but the "in the event" absolutely wrecked my brain, because I assumed it means _in the event payload_. Also, it seems to below a few lines higher, where `const delta =` is declared? Also `delta` seems like a strange name, since it's a duration, okay I'll stop now, sorry!
```suggestion
      // Use 1 as the difference in case startTimestamp === endTimestamp
```

Reply:
lol good catch on me moving the const but not changing the comment 🙃 

Also 100% agree re-reading the comment again I also got confused